src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/acl"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+)
+
+func NewACLCommand(mgr config.Manager) *cobra.Command {
```

Comment:
i think we should have security 

```
rpk security acl
rpk security service-account 
```

or alternatively

```
rpk acl --service-account "<username>:<password>" 
... and
rpk acl --kafka-acl "<service-account>:<topic-prefix-regex>"
```

i like the latter.